app/src/main/java/com/github/libretube/services/DownloadService.kt

```diff
@@ -74,17 +74,17 @@ class DownloadService : Service() {
             )
             if (downloadType in listOf(DownloadType.VIDEO, DownloadType.AUDIO_VIDEO)) {
                 videoDownloadId = downloadManagerRequest(
-                    getString(R.string.video),
+                    videoName,
                     getString(R.string.downloading),
                     videoUrl,
                     Uri.fromFile(
                         File(videoDownloadDir, videoName)
                     )
                 )
             }
-            if (downloadType in listOf(DownloadType.AUDIO, DownloadType.AUDIO_VIDEO)) {
+            else if (downloadType in listOf(DownloadType.AUDIO, DownloadType.AUDIO_VIDEO)) {
```

Comment:
Adding an else here will prevent video and audio to be downloaded with one request so we will need a different way to handle that one. I think it would be the easiest to give the second created notification just a low priority so that only the first one with the video name as title is visible.

Reply:
@Bnyro You're right! That's clumsy of me...😅 
The approach of using low priority(HIDDEN) notification needs the permission: _(android.permission.DOWNLOAD_WITHOUT_NOTIFICATION)_ 

If that's fine..Can I proceed with it?

Reply:
Hmm, another alternative would be to just use the video title for both, the video and the audio notification but that would suck too as the title would exist twice then.
So I think we'll need to live with the additional permission, so feel free to just go on :)